exercise_2/main.cpp

```diff
@@ -0,0 +1,59 @@
+#include <iostream>
+#include<fstream>
+#include "Graph.h"
+
+int const N = 8;
+
+void loadArrayFromFile(int(&arr)[N][N], std::string argv);
+
+void loadArrayFromFile(int (&arr)[N][N], std::string argv) {
+
+	
+	for (int i = 0; i < N; ++i)
+	{
+		for (int j = 0; j < N; ++j)
+		{
+			if (i != j)
+			{
+				arr[i][j] = -1;
+			}
+		}
+	}
+		std::ifstream in(argv);
```

Comment:
Indentation. This is not a python and compiler doesn't care about indentation, but for human it will be more readable.